mitodl/mitxpro
# feat: xPro blog

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes https://github.com/mitodl/hq/issues/909

#### What's this PR do?

- Adds a blog page in cms
- Fetches the blog by xPro author from https://curve.mit.edu and Adds a new page for blog at `/blog/`.

#### How should this be manually tested?

- Verify the designs https://projects.invisionapp.com/d/main#/console/22229357/474826812/preview
- Verify that the blog is fetched and cached for 24 hours.
- Verify that only MIT xPro author blog is fetched
- Verify the links
- verify the responsiveness
- Verify the blog CMS page is created using migrations and configure_wagtail
- Verify that the blog page changes reflect at `/blog`


#### Screenshots (if appropriate)
Laptop
![Opera Snapshot_2023-11-14_142800_xpro odl local](https://github.com/mitodl/mitxpro/assets/52656433/5f183ea4-3aa9-4373-8948-f8a06c384a65)
![Opera Snapshot_2023-11-14_142908_xpro odl local](https://github.com/mitodl/mitxpro/assets/52656433/7e19efbb-217b-4603-a304-4ebb22631e42)


Tablet
<img width="767" alt="Screen Shot 2023-11-14 at 2 30 32 PM" src="https://github.com/mitodl/mitxpro/assets/52656433/2d8d4a79-7a10-40fb-93f5-6df4b0eee27b">


Mobile:
<img width="431" alt="Screen Shot 2023-11-14 at 2 30 57 PM" src="https://github.com/mitodl/mitxpro/assets/52656433/84726353-f147-45be-8423-666b7f17d7fe">
